### DISCUSSION TITLE
TWITCH Instance 1 died: IndexError:list index out of range.

### DISCUSSION THREAD

**Discussion by Rogue963** [KevinBytesTheDust/CVAmp#301] [Q&A]
Hello, i'm getting this error, i have no idea what might be causing it i have my chrome driver latest version and it happens randomly, i can share logs privately if needed

**Comment by KevinBytesTheDust** [comment]
I am investigating this problem and provide a fix in the next version. What version do you use, GitHub or ko-fi?

**Reply by KevinBytesTheDust** [reply]
I have found the error. This happens in a previously unknown case where the auth_token makes it look like the user is logged in, but cvamp is unable to grab the user's name. I will log-warning this case, and we will find out what the possible cause is.

**Reply by Rogue963** [reply]
Appreciate it, i provided some logs in the Ko-fi i hope that will help, thanks for your attention
